modules/post-install-existing-local-zone-subnet.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: CONCEPT
+[id="post-install-existing-local-zone-subnet_{context}"]
+= Extend existing clusters to use AWS Local Zones
+
+If you want a Machine API to create an Amazon EC2 instance in a remote zone location, you must create a subnet in a Local Zone location. You can use any provisioning tool, such as Ansible or Terraform, to create subnets in the existing VPC.
+
+The following subsections include steps using CloudFormation templates. Considering the limitation of NAT Gateways in AWS Local Zones, CloudFormation templates support only public subnets. You can adapt it to your needs.
+
+You can modify the CloudFormation template to meet your requirements. Additionally, you can reuse the template to create public subnets for each edge location to where you need to extend your cluster.
```

Comment:
```suggestion
The following subsections include steps using CloudFormation templates. Considering the limitation of NAT Gateways in AWS Local Zones, CloudFormation templates support only public subnets. You can adapt it to your needs. Additionally, you can reuse the template to create public subnets for each edge location to where you need to extend your cluster.
```

Reply:
IMO, you might as well remove the duplicated statement that "you can adapt/modify the CloudFormation to meet your needs and just add that last sentence to the paragraph before. Just a suggestion!